### PR TITLE
The Biggest of Queries

### PR DESCRIPTION
**Pull request by ahandley**
Going to go fix all of the conflicts and merge them back in.  Coming shortly.

### CODE REVIEW

**Comment by ahandley** [comment]
Ok, @quoctruong , PTAL.  This should be it.

File: Google.PowerShell/packages.config [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
   <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net452" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net452" />
   <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net452" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net452" />
+  <package id="System.Interactive.Async" version="3.1.1" targetFramework="net452" />
   <package id="System.Management.Automation" version="6.1.7601.17515" targetFramework="net452" />
+  <package id="System.Net.Http" version="4.3.0" targetFramework="net452" />
   <package id="Zlib.Portable.Signed" version="1.11.0" targetFramework="net452" />
 </packages>
```
Comment (quoctruong) [inline Google.PowerShell/packages.config]: Can you add a new line here?

Comment (ahandley) [inline Google.PowerShell/packages.config]: Got it.

File: Google.PowerShell.IntegrationTests/packages.config [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <package id="Google.Apis.Bigquery.v2" version="1.20.0.667" targetFramework="net452" />
+  <package id="Google.Apis.Container.v1" version="1.20.0.662" targetFramework="net452" />
+  <package id="Google.Apis.Core" version="1.20.0" targetFramework="net452" />
+  <package id="Google.Cloud.BigQuery.V2" version="1.0.0-beta09" targetFramework="net452" />
+  <package id="log4net" version="2.0.3" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net452" />
+  <package id="System.Interactive.Async" version="3.1.1" targetFramework="net452" />
+  <package id="System.Net.Http" version="4.3.0" targetFramework="net452" />
+  <package id="Zlib.Portable.Signed" version="1.11.0" targetFramework="net452" />
+</packages>
```
Comment (quoctruong) [inline Google.PowerShell.IntegrationTests/packages.config]: I'm not sure if we need packages.config for the PowerShell.IntegrationTests project?

File: Google.PowerShell.Tests/packages.config [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+  <package id="Google.Apis.Storage.v1" version="1.20.0.753" targetFramework="net452" />
+  <package id="Google.Cloud.BigQuery.V2" version="1.0.0-beta09" targetFramework="net452" />
   <package id="log4net" version="2.0.3" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net452" />
   <package id="NUnit" version="2.6.4" targetFramework="net452" />
   <package id="NUnitTestAdapter" version="2.0.0" targetFramework="net452" />
+  <package id="System.Interactive.Async" version="3.1.1" targetFramework="net452" />
+  <package id="System.Net.Http" version="4.3.0" targetFramework="net452" />
   <package id="Zlib.Portable.Signed" version="1.11.0" targetFramework="net452" />
 </packages>
```
Comment (quoctruong) [inline Google.PowerShell.Tests/packages.config]: Add a line here.

File: Google.PowerShell.IntegrationTests/packages.config (hunk 2 above)
Comment (ahandley) [inline Google.PowerShell.IntegrationTests/packages.config]: should I just kill it then?

Comment (quoctruong) [inline Google.PowerShell.IntegrationTests/packages.config]: You can kill this, the app.config file and remove the extra packages that you added in Google.PowerShell.IntegrationTests/Google.PowerShell.IntegrationTests.csproj

File: Google.PowerShell.IntegrationTests/BigQuery/schema.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[{"Name":"Title","Type":"STRING"},
+{"Name":"Author","Type":"STRING"},
+{"Name":"Year","Type":"INTEGER"}]
```
Comment (quoctruong) [inline Google.PowerShell.IntegrationTests/BigQuery/schema.json]: New line here too.

File: Google.PowerShell.IntegrationTests/BigQuery/schema_mini.json [hunk 5]
```diff
@@ -0,0 +1 @@
+[{"Name":"Title","Type":"STRING"}]
```
Comment (quoctruong) [inline Google.PowerShell.IntegrationTests/BigQuery/schema_mini.json]: New line here.

**Pull request merged**